components/labels-and-certificates/patrimoine-vivant/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Section } from '#components/section';
+import { EAdministration } from '#models/administrations/EAdministration';
+
+export const LabelPatrimoineVivant = () => {
+  return (
+    <Section
+      title="Entreprises du patrimoine vivant"
+      sources={[EAdministration.MEF]}
+      id="patrimoine-vivant"
+    >
+      <p>
+        Cette structure est labellisée{' '}
+        <strong>« Entreprises du patrimoine vivant »</strong>.
```

Comment:
```suggestion
        <strong>« Entreprise du patrimoine vivant »</strong>.
```

Reply:
@rmonnier9 is there a reason this is not included ? The label should be in singular form https://www.entreprises.gouv.fr/espace-entreprises/s-informer-sur-la-reglementation/le-label-entreprise-du-patrimoine-vivant ?